irods/manager/data_object_manager.py

```diff
@@ -352,6 +358,7 @@ def make_FileOpenRequest(**extra_opts):
         use_get_rescinfo_apis = False
 
         if allow_redirect and conn.server_version >= (4,3,1):
+            print_debug()
```

Comment:
Should this be removed?

Reply:
I suppose I should really do my debug in another branch....

Reply:
removed